### PR TITLE
Update examples

### PR DESCRIPTION
**Pull request by LucieContamin**
- add metadata
- update target data format (aggregate into one parquet file format)
- update config files / article

### CODE REVIEW

**Review by elray1 — CHANGES_REQUESTED** [review]
I added a couple of comments, and added a PR [here](https://github.com/Infectious-Disease-Modeling-Hubs/example-complex-scenario-hub/pull/14) with some minor changes to the text.

The last two plots didn't work for me -- example output below.  I used the `fix_21` branch of hubVis.  Any thoughts about what might be going on?

image

File: hub-config/model-metadata-schema.json [hunk 1]
```diff
@@ -1,191 +1,144 @@
 {
-  "$schema": "http://json-schema.org/draft-07/schema",
-  "$id": "https://github.com/Infectious-Disease-Modeling-Hubs/schemas/blob/main/v0.0.0.9/model-schema.json",
-  "description": "This is the schema of the model metadata file, please refer to [model-metadata](https://hubdocs.readthedocs.io/en/latest/format/model-metadata.html) documentation for more information.",
-  "type": "object",
-  "properties": {
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "Schema for Modeling Hub model metadata",
+    "description": "This is the schema for model metadata files, please refer to https://github.com/covid19-forecast-hub-europe/covid19-forecast-hub-europe/wiki/Metadata for more information.",
```
[the remark this answers is not in the document]
Comment (LucieContamin) [reply hub-config/model-metadata-schema.json:4]: Thank you for noticing! I missed it. I will update it to the README in the example repo.

File: model-metadata/hubcomp_example.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+team_name: "Hub Complex Example"
+team_abbr: "hubcomp"
+model_name: "Example Model output"
+model_abbr: "examp"
```
[the remark this answers is not in the document]
Comment (LucieContamin) [reply model-metadata/hubcomp_example.yaml]: Thanks for catching that! I fix the file name!

**Comment by LucieContamin** [comment]
@elray1 . Thank you very much for your review.

I update the metadata schema `description` information to update the link and fix the `.yaml` file name. Thank you again!

For the plots, it was cause by the empty `age_group` columns. In a past version of the file, I was removing it. So, I updated the code here to remove the empty columns, but I also updated the hubvis `fix_21` branch (PR [here](https://github.com/Infectious-Disease-Modeling-Hubs/hubVis/pull/23)) to automatically remove empty columns and returns a warning, to avoid that problem in the future.

**Review by elray1 — APPROVED** [review]
Thanks, this all looks good to me!

**Pull request merged**